007_team_5_agent_design/007_team_5_agent_design.tex

```diff
@@ -1 +1,331 @@
+\end{itemize}
+The leadership value is calculated by:
+\[ \texttt{leadership} = x - \texttt{selfishness} - \texttt{agentFloor}\]
+The variable $x$ is a random number between 3 and 12, inclusive. If $leadership$ is greater than the agent's leadership threshold, then the agent becomes a leader and proposes a treaty.
+
+The proposed treaty is always that agents whose HP is greater than or equal to a certain value should always leave 100\% of the food on the platform, and always lasts for five days. The threshold value for HP is calculated using Equation \ref{eq:team5-treaty-hp-threshold}. Note that in this equation, \texttt{WeakLevel} refers to the threshold between the ``critical'' and ``weak'' HP states.
+
+\begin{equation}\label{eq:team5-treaty-hp-threshold}
+    \texttt{hpThreshold} = \texttt{aimHP} - \frac{\texttt{aimHP} - \texttt{WeakLevel}}{10}\times\left(\texttt{leadership} - \texttt{leadershipThreshold}\right)
+\end{equation}
+
+\section{Experimentation}\label{sec:team5-experimentation}
+This section focuses on how the agent's performance varies with changes in its environment.
+
+\subsection*{Homogeneous Tower Experiments}
+To evaluate the self-organisation abilities of the Team 5 agent, simulations were run with the agent in variations of a homogeneous tower (i.e. a tower that contains agents of only one type). The agent has been evaluated on three main metrics:
+\begin{enumerate}
+    \item The number of deaths
+    \item The average utility
+    \item Whether self-organisation was achieved.
+\end{enumerate}
+We have defined self-organisation as when all agents take only what they need to survive and there are little to no deaths beyond a certain point in time.
+
+\subsubsection*{Food Per Agent On Platform}
+The baseline experiment for testing was to vary the amount of food per agent available on the platform; we expect that more food being available will result in fewer deaths. In this experiment, simulations were run for 100 simulation days, with a reshuffle period of 7 days and 50 agents in the tower and repeated three times, with the average of these runs taken. Figure \ref{fig:team5-deaths-utility-food-per-agent} shows that the number of deaths decreases and average utility increases as we increase the amount of food available.
+
+When the ratio of food to agent is greater than 9, the agents are able to self-organise within 100 days, and typically do so within 70 days. When food availability is less than this, the agents did not achieve self-organisation, however interestingly, it was observed that more treaties were accepted than rejected. This shows that the agents recognise theneed to self-organise and are willing to make attemps to do so. It is possible that they did not achieve self-organisation because agents died too quickly to reach agreements among enough agents for this to be successful.
```

Comment:
```suggestion
When the ratio of food to number of agents was greater than 9, the agents were able to self-organise within 100 days, and typically do so within 70 days. When food availability was less than this, the agents did not achieve self-organisation. However, it was interesting to observe that more treaties were accepted than rejected. This showed that the agents recognised the need to self-organise and were willing to make attempts to do so. It is possible that they did not achieve self-organisation because agents died too quickly to reach agreements among enough agents for this to be successful.
```